src/libs/SidebarUtils.ts

```diff
@@ -146,7 +147,10 @@ function getOrderedReportIDs(
     const isInDefaultMode = !isInGSDMode;
     const allReportsDictValues = Object.values(allReports);
     // Filter out all the reports that shouldn't be displayed
-    const reportsToDisplay = allReportsDictValues.filter((report) => ReportUtils.shouldReportBeInOptionList(report, currentReportId, isInGSDMode, betas, policies, allReportActions, true));
+    const reportsToDisplay = allReportsDictValues.filter((report) =>
+        // TODO: Talk with Fabio about that
```

Comment:
@kubabutkiewicz @fabioh8010 don't forget to talk :)

Reply:
We did ! 😄